### PR TITLE
Added support for Django 3.2

### PR DESCRIPTION
**Pull request by hramezani**

### CODE REVIEW

**Review by jezdez — APPROVED** [review]
LGTM, maybe we should get rid of py3.5 as well?

File: setup.py [hunk 1]
```diff
@@ -30,6 +30,7 @@ def read(*parts):
         "Framework :: Django :: 2.2",
         "Framework :: Django :: 3.0",
```
Comment (carltongibson) [inline setup.py:31]: Django 3.0 is now EOL. Don't know if that's out of scope for this PR.

Comment (hramezani) [reply setup.py:31]: I will manage it in separate PR

**Comment by hramezani** [comment]
> LGTM, maybe we should get rid of py3.5 as well?

Maybe we need to keep Py3.5 support for Django2.2

**Comment by jezdez** [comment]
Problem is that Python 3.5 is EOL already and doesn't even receive security fixes anymore. It's not sensible to support Python versions that have stopped being supported from upstream.

**Comment by carltongibson** [comment]
@jezdez People have shouted at me for saying that. 😀
(I agree but...)

**Comment by hramezani** [comment]
Ok, I will create a patch to remove support for python 3.5

**Comment by jezdez** [comment]
> @jezdez People have shouted at me for saying that. 😀
> (I agree but...)

I feel for you being shouted at, I've been there.

**Comment by claudep** [comment]
It's nothing serious enough to shout at anyone, but you also have to understand that some people are looking for long-term stability (going even to the extra effort of backporting security to no-longer-supported releases), and I think both parties should respect one another, WITHOUT SHOUTING :-)

**Comment by jezdez** [comment]
> It's nothing serious enough to shout at anyone, but you also have to understand that some people are looking for long-term stability (going even to the extra effort of backporting security to no-longer-supported releases), and I think both parties should respect one another, WITHOUT SHOUTING :-)

True, you've probably seen many of those discussions over the years, so I'm happy you're chiming in! No shouting needed :)

There is sadly no way to handle custom long-term support requests for Jazzband projects other than on the individual member level at the moment. If in doubt, please feel free to reach out to me (or the project leads) if there is interest in working something out.

**Pull request merged**